lib/master_manipulator/version.rb

```diff
@@ -1,5 +1,5 @@
 module MasterManipulator
   module Version
-    STRING = '2.0.0'
+    STRING = '2.1.0'
```

Comment:
Just confirmed that it does... was looking in my old PR where Kurt said as much.